README.md

```diff
@@ -1 +1,110 @@
-# BC Vaccine Card-Android
\ No newline at end of file
+# BCVAX-Android
+
+#### BC Vaccine Card Verifier App is used to scan the QR generated from HealthGateway app and shows the vaccination status of a person. This is for Businesses from the province of British Columbia.
+
+## Table of Contents
+
+- [Project Resources](#markdown-header-project-resources)
+- [Architecture](#markdown-header-architecture)
+  - [Folder Structure](#markdown-header-folder-structure)
+  - [Libraries](#markdown-header-libraries)
+- [Configuration](#markdown-header-configuration)
+  - [Build Variants](#markdown-header-build-variants)
+  - [Environment Variables](#markdownpheader-environment-variables)
+  - [APIs](#markdown-header-apis)
+- [Deployment](#markdown-header-deployment)
+  - [Versioning](#markdown-header-versioning)
+  - [Internal](#markdown-header-internal)
+  - [External](#markdown-header-external)
+- [Contributors](#markdown-header-contributors)
+
+## Project Resources
+---
+
+List any external project resources. Possible resources include:
```

Comment:
This line is not required. Please remove it.

Reply:
changed